htdocs/projet/card.php

```diff
@@ -1023,7 +1023,7 @@ function set_usage_task() {
 
 					function set_usage_bill_time() {
 						console.log("set_usage_bill_time");
-						if (jQuery("#usage_bill_time").prop("checked")) {
+						if (jQuery("#usage_bill_time").prop("checked") || jQuery("#usage_task").prop("checked") ) {
```

Comment:
Why adding the condition on usage task ?
To create bill, the project role we need is usage_bill_time.

Reply:
Because if you just have task usage checked, the budget field is hidden as it have classusebilltime class